src/js/components/AlignmentChoiceButton.js

```diff
@@ -1,18 +1,40 @@
 import React, { Component } from 'react';
-import { saveAlignment } from '../actions/nextStepCharAction';
+import { connect } from "react-redux";
+import { bindActionCreators } from 'redux';
+import * as actionCreators from '../actions/nextStepCharAction';
+import "../../styles/attributes.scss";
 
-require("../../styles/attributes.scss");
+const AlignmentChoiceButton = ({alignment, actions, lastStep}) => {
+    let cssClass = getProperActiveElement(alignment, lastStep);
 
-export default class AlignmentChoiceButton extends Component {
-    saveAlignment() {
-        return this.props.dispatch(saveAlignment(this.props.alignment));
+    return (
+        <button class={cssClass} onClick={() => saveAlignmentToStore(alignment, actions)}>
```

Comment:
do not use this synatx for onClick
Basically its recommended to avoid creating function in onClick, because this function is created each time when render occurs,
so its better to write `onclick={onButtonClick(alignment, actions)}`
and add `onButtonClick=(alignment, actions) => (event)  => ....` to class property